great_expectations/rule_based_profiler/parameter_builder/numeric_metric_range_multi_batch_parameter_builder.py

```diff
@@ -181,7 +181,11 @@ def _build_parameters(
             )
 
         metric_computation_result: Dict[
-            str, Union[Union[np.ndarray, List[Union[Any, Number]]], Dict[str, Any]]
+            str,
```

Comment:
@jcampbell This is an incorrect change, because metrics do not always return numbers.  This is at the core of the architecture enhancement idea I mentioned in a meeting on 11/30/2021 -- to enable the desired flexibility.

Reply:
Ok....so net, I think you're suggesting leaving as is?